crates/primitives/src/buf.rs

```diff
@@ -78,6 +125,15 @@ impl From<XOnlyPublicKey> for Buf32 {
     }
 }
 
+// NOTE: we cannot do `ZeroizeOnDrop` since `Buf32` is `Copy`.
+#[cfg(feature = "zeroize")]
+impl Zeroize for Buf32 {
+    #[inline]
+    fn zeroize(&mut self) {
+        self.0.zeroize();
+    }
+}
+
 // 64-byte buf, useful for schnorr signatures
 #[derive(Copy, Clone, Eq, PartialEq, Ord, PartialOrd, Hash)]
 pub struct Buf64(pub [u8; 64]);
```

Comment:
Why not implement `Zeroize` for this as well, for consistency?

Reply:
Oh I forgot this one!
Nice catch!

Reply:
Done in 0e362baa78432aa47855c2d5126aa5c83490e0de